tensorflow/core/ops/io_ops.cc

```diff
@@ -582,6 +582,15 @@ REGISTER_OP("ReadFile")
 Reads and outputs the entire contents of the input filename.
 )doc");
 
+REGISTER_OP("WriteFile")
+    .Input("filename: string")
+    .Input("contents: string")
+    .Output("output: int32")
+    .SetShapeFn(shape_inference::ScalarShape)
+    .Doc(R"doc(
+Writes contents to the end of input filename. Creates file if not existing.
```

Comment:
Why does this op need an output?  Who is going to consume the output?

Should we just have it produce no outputs and do an OP_REQUIRES(WriteStringToFile...) in its implementation?